GameOfNim/Program.cs

```diff
@@ -0,0 +1,118 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+var numberOfMatches = 24;
+var isPlayerOneTurn = true;
+bool isHardDifficulty = false;
```

Comment:
inconsistent use of `var` and explicit data type. Either is fine. Both is not good :D